uaclient/http/__init__.py

```diff
@@ -398,6 +398,8 @@ def readurl(
             json_dict = json_body
         elif isinstance(json_body, list):
             json_list = json_body
+        else:
+            LOG.warning("unexpected JSON response: %s", str(json_body))
```

Comment:
There is just a typo in the commit message of this one. It says "we will **not** log..." but should say "we will now log..."